src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -190,6 +190,14 @@ export class BrowsingContextImpl {
     return this.#parentId === null;
   }
 
+  get top(): BrowsingContextImpl {
```

Comment:
BrowsingContextStorage has a `findTopLevelContext` method already. Please reuse it.

Reply:
Can there be more than one top-level context? In theory, it could be a forest, no?

Reply:
Nevermind, my interpretation of the function was wrong.

Reply:
Hmm, I think I'd rather keep this method and remove `findTopLevelContext`. `findTopLevelContext` is not used anywhere and it's far slower than this `BrowsingContextImpl.top`.